index.js

```diff
@@ -1,6 +1,6 @@
 /*** EventWatcher Z-Way HA module *******************************************
 
-Version: 1.04
+Version: 1.05
```

Comment:
Please don't increment the version. I have scripts for building releases that will get confused by manual changes